test/cqlpy/test_lwt.py

```diff
@@ -83,13 +83,18 @@ def test_lwt_empty_clustering_range(cql, table1):
 # so if a batch has both a IF EXISTS and IF NOT EXISTS on the same row, they
 # can't possibly both be true, so this batch is guaranteed to fail
 # regardless of the data. Cassandra detects this specific conflict, and
-# prints an error instead of silently failing the batch.
-# Reproduces #13011.
-@pytest.mark.xfail(reason="issue #13011")
-def test_lwt_with_batch_conflict_1(cql, table1):
+# prints an error instead of silently failing the batch, but in ScyllaDB
+# we considered this check to be inconsistent and unhelpful, and
+# decided not to implement it, and this case is treated as a normal batch
+# failure (not all conditions are true), not an error. See discussion in #13011.
+# The test is marked scylla_only because it will fail on Cassandra which will
+# report an error, not a batch failure.
+def test_lwt_with_batch_conflict_1(cql, table1, scylla_only):
     p = unique_key_int()
-    with pytest.raises(InvalidRequest, match='Cannot mix'):
-        cql.execute(f'BEGIN BATCH DELETE FROM {table1} WHERE p={p} AND c=1 IF EXISTS; INSERT INTO {table1}(p,c,r) VALUES ({p},1,2) IF NOT EXISTS; APPLY BATCH;')
+    rs = list(cql.execute(f'BEGIN BATCH DELETE FROM {table1} WHERE p={p} AND c=1 IF EXISTS; INSERT INTO {table1}(p,c,r) VALUES ({p},1,2) IF NOT EXISTS; APPLY BATCH;'))
```

Comment:
I think the `list()` isn't necessary here (the iterator can be iterated without being materialized first). But I guess it doesn't hurt either. But I see you copied this line from a pre-existing function, so fine.